stock_orderpoint_generator/models/orderpoint_template.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+# © 2012-2016 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+from openerp import api, fields, models
+
+
+class OrderpointTemplate(models.Model):
+    """ Template for orderpoints
+
+    Here we use same model as stock.warehouse.orderpoint but set product_id
+    as non mandatory as we cannot remove it. This field will be ignored.
+
+    This has the advantage of ensuring that the order point
+    and the order point template have the same fields.
+
+    _table is redefined to separate templates from orderpoints
+    """
+    _name = 'stock.warehouse.orderpoint.template'
+
+    _inherit = 'stock.warehouse.orderpoint'
+    _table = 'stock_warehouse_orderpoint_template'
```

Comment:
Hi @lreficent,
Looks like that is not necessary as it is the same as the new class name, I just maintain the code as it was in the backport module.